pkgs/by-name/kr/kryptor/update.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -i bash -p nix cacert curl jq common-updater-scripts
+set -euo pipefail
+
+new_version="$(curl -s "https://api.github.com/repos/samuel-lucas6/Kryptor/releases?per_page=1" | jq -r '.[0].tag_name | ltrimstr("v")')"
```

Comment:
Is there a reason not to just use `nix-update-script` here?

Reply:
Using `passthru.updateScript = nix-update-script { };` won't update `deps.nix`. Am I missing some argument for `nix-update-script` or that should be used inside the custom `update.sh`?

Reply:
I was thinking you'd still have the custom `update.sh` but just call `nix-update` before `fetch-deps`.

It would just save having to deal with the github API, etc.  That's assuming the default behaviour of `nix-update` is okay for this package.

Reply:
Thanks a lot, the update script is much cleaner now, I was surprised I haven't seen this pattern, especially for dotnet update scripts.